scripts/test_commands.sh

```diff
@@ -12,6 +18,12 @@ echoGreen(){
     echo -e "\e[1;32m$1"
 }
 
+neuPath="node $(pwd)/../../bin/neu.js"
```

Comment:
I vote for the previous approach of running `neu` with `npm link` due to several reasons like simplicity. Perhaps, you can check with @pathange-s and agree on the most suitable approach since he initially implemented the script. If you both agree to use the `npm link` approach, please use `neu` instead of `$neuPath` Thanks :tada:

Reply:
I suggest connecting me over discord for quicker progress.
> Previously, the test script test_commands.sh used the already installed neu command to invoke neu CLI. This commit makes the test script use current source code to run tests, making it more convenient to test in local development.
Also, the current implementation uses latest source code for testing. You can check with what `npm link` does once for better understanding. If you still have any queries, consider tagging me for a discussion.